pkgs/development/libraries/spice/default.nix

```diff
@@ -37,11 +39,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "spice";
-  version = "0.15.0";
+  version = "0.15.1";
 
-  src = fetchurl {
+  src = fetchzip {
```

Comment:
Why use `fetchzip`? That will unnecessarily create extra store entry for the unpacked source.

Reply:
I prefer having the source files directly than some tarball that needs to be unpacked first. Makes unpackPhase faster and more efficient, especially with reflinks.

The tarball will be gc'd anyways.

Reply:
See https://discourse.nixos.org/t/github-archive-checksums-may-change/25108 for another reason to prefer specialised fetchers over `fetchurl`.